### PR TITLE
More logs

### PR DESCRIPTION
**Pull request by davidlevanon**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,7 +29,14 @@ function dynalite(options) {
 	options = options || {}
 	var server, stores = {}, requestHandler = httpHandler.bind(null, stores, options)
 
-	logger = bunyan.createLogger("dynalite_log", __dirname, options)
+	logger = dynaliteLogger.createLogger("dynalite_log", __dirname, options)
+	
+	if (options.memoryStats) {
+		setInterval(function() {
+			logger.info("Process Memroy Stats: " + stringify(process.memoryUsage()));
```
Comment (dor-levi) [inline index.js]: Can `stringify(process.memoryUsage())` fail?

Probably should surround this with try/catch

File: index.js [hunk 2]
```diff
@@ -340,6 +355,93 @@ function httpHandler(stores, options, req, res) {
...
+	return {tableName:tableName, details:details}
+}
+
+function logAction(action, data, preData, actionTimeMs) {
+	var details = preData.details;
+	var tableName = preData.tableName;
+	
+	try {
+		if (data) {
+			if (action == "createTable") {
```
Comment (dor-levi) [inline index.js]: I don't fully understand the purpose of this if statement

File: index.js [hunk 3]
```diff
@@ -340,6 +355,93 @@ function httpHandler(stores, options, req, res) {
...
+			} else if (action == "describeTable") {
+				details = "status: " + data.Table.TableStatus;
+			}
+		}
+	} catch (e) {
+		tableName = "error";
```
Comment (dor-levi) [inline index.js]: Not sure tableName should be overridden here

File: logger/index.js [hunk 4]
```diff
@@ -1,71 +1,62 @@
...
+					var msg = util.format("%s %s %s", 
+						pad(formattedTime, 20),
+						pad(options.level.toUpperCase(), 6),
+						options.message);
+					
+					return msg
+				},
+				filename: fullPath,
+				json: false,
+				level: "debug"
```
Comment (dor-levi) [inline logger/index.js:48]: Do we want level param to be configurable?

**Review by dor-levi — CHANGES_REQUESTED** [review]

File: logger/index.js (hunk 4 above)
Comment (davidlevanon) [reply logger/index.js:48]: We don't always use logger in our "dynalite". But, when we use it we put "debug" to see everything. 

Look at the "if (!opts.verbose) {" in the beginning of the method.

Comment (dor-levi) [reply logger/index.js:48]: I've seen it, still wondered if we want this more configurable.
No is a valid answer :)

File: index.js [hunk 5]
```diff
@@ -340,6 +360,91 @@ function httpHandler(stores, options, req, res) {
...
+			tableName = Object.keys(data.RequestItems);
+		}
+		
+		details = stringify(data.RequestItems);
+	} else if (action == "deleteItem") {
+		tableName = data.TableName;
+		details = stringify(data.Key)
+	} else if (action == "scan") {
+		tableName = data.TableName;
+		details = stringify(data)
```
Comment (dor-levi) [inline index.js]: Missing `;`

File: index.js [hunk 6]
```diff
@@ -340,6 +360,91 @@ function httpHandler(stores, options, req, res) {
...
+	} else {
+		details = stringify(data);
```
Comment (dor-levi) [inline index.js:415]: We might want to consider

```
if (data.TableName)
{
    tableName = data.TableName;
}
```

So we'll catch any actions we missed

File: index.js [hunk 7]
```diff
@@ -340,6 +360,91 @@ function httpHandler(stores, options, req, res) {
...
+	
+	return {tableName:tableName, details:details}
+}
+
+function logAction(action, data, preData, actionTimeMs) {
+	var details = preData.details;
+	var tableName = preData.tableName;
+	
+	try {
+		if (data) {
```
Comment (dor-levi) [inline index.js]: can be `if ((action == "describeTable") && (data) && (data.Table)) {` to save some indentation and crash possibilities

**Review by dor-levi — CHANGES_REQUESTED** [review]

**Review by dor-levi — APPROVED** [review]

**Pull request merged**